File: Backend/BaseMgtService/Senior.AgileAI.BaseMgt.Application/DTOs/CreateOrganizationDTO.cs
using Microsoft.AspNetCore.Http;

namespace Senior.AgileAI.BaseMgt.Application.DTOs
{
    public class CreateOrganizationDTO
    {
        public Guid UserId { get; set; } //it will used as the manger id
        public required string Name { get; set; }
        public required string Description { get; set; }
        public IFormFile? LogoFile { get; set; }
    }
}

File: agilemeets_mobile/lib/data/models/organization/create_organization_dto.dart
class CreateOrganizationDTO {
  final String userId;
  final String name;
  final String description;
  final String? logo;

  CreateOrganizationDTO({
    required this.userId,
    required this.name,
    required this.description,
    this.logo,
  });

  Map<String, dynamic> toJson() {
    return {
      'userId': userId,
      'name': name,
      'description': description,
      if (logo != null) 'logo': logo,
    };
  }
} 